### PR TITLE
Add Gotify as notification backend

### PR DESCRIPTION
**Pull request by mad73923**
This adds Gotify as backend

### CODE REVIEW

File: impf/alert.py [hunk 1]
```diff
@@ -146,4 +148,19 @@ def pushover_send(message: str) -> None:
...
-    logger.debug(r)
\ No newline at end of file
+    logger.debug(r)
+
+@alert_resilience
+def gotify_send(message: str) -> None:
+    url = f'{settings.GOTIFY_URL}/message'
+    params = (
+        ('token', f'{settings.GOTIFY_APP_TOKEN}')
+    )
```
Comment (alfonsrv) [inline impf/alert.py:158]: Does this work? Never seen tuples used as params, heh. Usually params should be a `dict` – e.g.

```python
params = {
    'token': settings.GOTIFY_APP_TOKEN
}
```

Comment (mad73923) [reply impf/alert.py:158]: Yes, tested with the --alerts argument. works fine!

**Comment by alfonsrv** [comment]
Alright looks good. Merging to main – thank you for contributing.

**Comment by sethslife** [comment]
Fag

**Pull request merged**